ui/vic-ui-h5c/vic-service/src/main/java/com/vmware/vic/model/VirtualContainerHostVm.java

```diff
@@ -78,7 +86,12 @@ private void processExtraConfig(ArrayOfOptionValue ovs) {
 						}
 					}
 					_clientIp = sb.toString();
-					break;
+					continue;
+				}
+
+				if (EXTRACONFIG_DOCKER_PERSONALITY_ARGS_KEY.equals(key)) {
+				    _isUsingTls = ((String)ov.getValue()).indexOf("2376") > -1;
```

Comment:
It is a fixed port used by Docker engine server from the backend. It uses either port 2375 for  no TLS, or 2376 for TLS. For your reference, see the following screenshot showing part of the VMX config for a VCH VM.

![image](https://cloud.githubusercontent.com/assets/3834071/26071333/540ed33a-396d-11e7-8ce8-804e0b9616ba.png)